src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,66 @@
+package duke.ui;
+
+import java.util.Scanner;
+import duke.exception.DukeException;
+
+public class Ui {
+    private Scanner scan = new Scanner(System.in);
+    private String dukeLogo = " ____        _        \n" + "|  _ \\ _   _| | _____ \n" + "| | | | | | | |/ / _ \\\n"
+            + "| |_| | |_| |   <  __/\n" + "|____/ \\__,_|_|\\_\\___|\n";
+
+    /**
+     * Prints out standard welcome message.
+     */
+    public void welcomeUser() {
+        System.out.println("Hello, I am \n" + dukeLogo + "your personal buddy. What's up?\n"
+                + "____________________________________________________________\n");
+    }
+
+    /**
+     * Prints out standard farewell message.
+     */
+    public void farewellUser() {
+        divider("Aww, see you again soon!");
+        return;
```

Comment:
I think the `return;` statement here could be removed as this function has a return type of `void`.